scipy/optimize/tests/test_optimize.py

```diff
@@ -2200,6 +2200,14 @@ def func(z, *params):
                           match=r'Either final optimization did not succeed'):
             optimize.brute(func, self.rranges, args=self.params, disp=True)
 
+    def test_coerce_args_param(self):
+        # optimize.brute should coerce non-iterable args to a tuple.
+        def f(x, *args):
+            return x ** args
+
+        resbrute = optimize.brute(f, (slice(-4, 4, .25),), args=2)
+        assert_almost_equal(resbrute, 0)
```

Comment:
```suggestion
        assert_allclose(resbrute, 0)
```
From the [documentation of `assert_almost_equal`:](https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_almost_equal.html)
![image](https://user-images.githubusercontent.com/6570539/174417206-e29114fd-0106-4c80-af83-1dc34bae52b8.png)

I understand that the intent may have been to match the style of existing code, but we typically use `assert_allclose` in new code even if the old code uses `assert_almost_equal`.

Reply:
@mdhaber thanks, I learned something today! :-)